api/registry/api.py

```diff
@@ -193,6 +196,18 @@ def community_requires_signature(_):
     return False
 
 
+def get_scorer_id(payload: SubmitPassportPayload) -> str:
+    scorer_id = ""
+    if payload.scorer_id:
+        scorer_id = payload.scorer_id
+    elif payload.community != "Deprecated":
```

Comment:
Should we also check to make sure community isn't empty?

Maybe 203 should be
`elif payload.community and payload.community != "Deprecated":`